scripts/historical_exporter.py

```diff
@@ -24,6 +24,10 @@ def main():
         # end: yvUSDC vault January-20-2022 06:10:45 AM +-6 UTC
         end = datetime(2022, 1, 20, tzinfo=timezone.utc)
         data_query = 'yearn_vault{network="GNO"}'
+    elif Network(chain.id) == Network.Arbitrum:
+        # end: yvUSDC 4.3 Sep-05-2021 09:05:50 PM +UTC
+        end = datetime(2021, 9, 05, tzinfo=timezone.utc)
+        data_query = 'yearn_vault{network="ARRB"}'
```

Comment:
`network="ARRB"`
not 100% sure this is correct but its the network label it has below